### PR TITLE
Don't try to build the app when using only the `-h` or `--help` flags on the `run` command

### LINKED ISSUE #341: Running vapor run -h or --h fails when it cannot detect a manifest file within the pwd
### Issue

When running `vapor run -h` or `vapor run --h` to see the list of available commands, the command fails if it does not detect a manifest file within the current working directory. This also applies to any of the sub commands under `run`. Fortunately, this seems to be the only command for which this happens: i.e. the help pages for all other vapor commands are visible normally.
Screen Shot 2020-08-21 at 3 05 46 PM
### Expected behavior

Checking the Vapor help section for any command should simply display its complete help section, without depending on the presence of any external sources.

### PR DESCRIPTION
**Pull request by tkrajacic**
Also add an explanatory line to the output with the equivalent swift package manager command.

image

Fixes #341

### CODE REVIEW

**Review by marzvrover — COMMENTED** [review]
I believe this would read more easily if instead of an `if/else` you used `guard` before the `do/catch`.

```swift
guard context.input.arguments.count > 1 else {
    context.console.output("\(self.help)")
    return
}

do {
…
```

File: Sources/VaporToolbox/Run.swift [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Foundation
 
 // Generates an Xcode project
 struct Run: AnyCommand {
-    let help = "Runs an app from the console."
+    let help = "Runs an app from the console.\nEquivalent to `swift run --enable-test-discovery Run`."
```
Comment (0xTim) [inline Sources/VaporToolbox/Run.swift:6]: I don't think this should be wrapped in a markdown code block as Terminal won't render it like that

**Review by 0xTim — APPROVED** [review]
Minor nit about the formatting otherwise LGTM

File: Sources/VaporToolbox/Run.swift (hunk 1 above)
Comment (tkrajacic) [reply Sources/VaporToolbox/Run.swift:6]: I used backticks, as they signify commands that are executed in shells (which predates Markdown by decades). No Markdown here. 🤣

Do you want me to switch it for single quotes?

Comment (0xTim) [reply Sources/VaporToolbox/Run.swift:6]: Nah I'm happy with it

**Comment by tanner0101** [comment]
These changes are now available in [18.3.1](https://github.com/vapor/toolbox/releases/tag/18.3.1)

**Pull request merged**